crates/bevy_render/src/spatial_bundle.rs

```diff
@@ -27,6 +30,8 @@ pub struct SpatialBundle {
     pub transform: Transform,
     /// The global transform of the entity.
     pub global_transform: GlobalTransform,
+
+    pub _marker: ToRenderWorld,
```

Comment:
its used for fog_volume or anything utilize spatial_bundle in render world, ToRenderWorld is a explicit component indicate that its entity needs to sync with render world , it maintained by WorldSyncPlugin<ToRenderWorld>

Reply:
Feels a bit weird adding this to spatial bundle rather than the specific bundles that need to be extracted. Users often use SpatialBundle when building their own entities, but they might not need to be extracted to the RenderWorld.